icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -241,11 +241,21 @@ cd $ICU4J_ROOT
 ant all 2>&1 | tee /tmp/icu4j-oldData-antAll.txt
 ant check 2>&1 | tee /tmp/icu4j-oldData-antCheck.txt
 
-# 12. Now build the new data and test data for ICU4J
+# 12. Transfer the data to ICU4J.
+# 12a. You must first configure ICU in order to add the unicore data.
+
+cd $ICU4C_DIR/source
+ICU_DATA_BUILDTOOL_OPTS=--include_uni_core_data ./runConfigureICU Linux
+
+# 12b. Now buil the jar files.
```

Comment:
typo "buil"

Reply:
I noticed this; thought wasn't bad enough to justify another review cycle